contracts/events/LoanOpeningsEvents.sol

```diff
@@ -13,6 +13,8 @@ pragma solidity 0.5.17;
  * This contract contains the events for loan openings operations.
  * */
 contract LoanOpeningsEvents {
+	event ContractReplaced(address indexed changedBy, address indexed newTargetAddr, bytes32 indexed module);
```

Comment:
```suggestion
	event ProtocolModuleContractReplaced(address indexed changedBy, address indexed newTargetAddr, bytes32 indexed module);
```
pls mind naming as it is used by FE and BE too

Reply:
Renamed and added prevModuleContractAddress